server/api/bot/submit.post.ts

```diff
@@ -5,7 +5,16 @@ const submitBotCodeSchema = z.object({
   code: z.string(),
 });
 
+const rateLimit = new Map<string, number>();
```

Comment:
супер мелкий nit: мы капитализируем глобальные константы в проекте

```suggestion
const RATE_LIMIT_MAP = new Map<string, number>();
```